src/Common/src/Interop/OSX/System.Security.Cryptography.Native.Apple/Interop.Ssl.cs

```diff
@@ -579,56 +590,76 @@ internal static void SslSetTargetName(SafeSslHandle sslHandle, string targetName
 
         internal static unsafe void SslCtxSetAlpnProtos(SafeSslHandle ctx, List<SslApplicationProtocol> protocols)
         {
-            byte[] buffer = ConvertAlpnProtocolListToByteArray(protocols);
-            int osStatus;
-
-            fixed (byte* b = buffer)
+            SafeCreateHandle cfProtocolsRefs = null;
+            SafeCreateHandle[] cfProtocolRefs = null;
```

Comment:
Maybe the first one should be `cfProtocolsRef`, or `cfProtocolsArrayRef`.  I was not noticing the interior `s` difference in the `else` case, and was greatly confused.